<launch>
 	<node pkg="lidar_to_point_cloud_transformer" type="lidar_to_point_cloud_transformer" name="lidar_to_point_cloud_transformer" output="screen">
    	<param name="laser_scan_topic" type="string" value="/multisense/lidar_scan" />
    	<param name="fixed_frame" type="string" value="world" />
		<param name="target_frame" type="string" value="head" />
		<param name="base_frame_id" type="string" value="/multisense/motor" />
    	<param name="rotating_frame_id" type="string" value="/multisense/hokuyo_link" />
    	<param name="limit_angle" type="double" value="3.14159265" />
    	<rosparam command="load" file="$(find lidar_to_point_cloud_transformer)/parameters/atlas_laser_scan_filter_chain.yaml" />
    </node>
</launch>